python/ray/serve/run_script.py

```diff
@@ -109,6 +110,14 @@ def main():
         serve.shutdown()
         sys.exit()
 
+    except Exception:
+        traceback.print_exc()
+        cli_logger.info(
+            "Received unexpected error, see logs for more details. Shutting down..."
```

Comment:
Might be better to use `cli_logger.error`, and if possible we should give a hint about which logs to look at. (Will they be in the terminal, or will they be in a certain file at `/tmp/ray/session_latest/logs`? Based on the test I guess they're printed to the terminal)

Reply:
Thanks for the suggestion, addressed!